OpenSim/Simulation/Model/PhysicalFrame.h

```diff
@@ -48,8 +52,30 @@ class Body;
 */
 
 class OSIMSIMULATION_API PhysicalFrame : public Frame {
-    OpenSim_DECLARE_CONCRETE_OBJECT(PhysicalFrame, Frame);
+    OpenSim_DECLARE_ABSTRACT_OBJECT(PhysicalFrame, Frame);
 
+public:
+    //==============================================================================
+    // PROPERTIES
+    //==============================================================================
+    /** @name Property declarations
+    These are the serializable properties associated with a PhysicalFrame.
+    NOTE: Thes properties used to be members of Body and were moved up the hierarchy
+    with the introduction of Frames.
+
+    TODO: Both VisibleObject and the WrapObjectSet should NOT be properties
+    of the PhysicalFrame. This is an itermediate solution as we integrate Frames 
+    use into the OpenSim API. These properties should be their own components with
+    Connectors to the PhysicalFrames they attach to. This must be addressed prior
+    to OpenSim 4.0 release. - aseth */
```

Comment:
I think this comment can also be removed from the code. I think it'd be better to keep track of TODO's as github issues. Say we forget to remove this comment from the code even after it's been addressed; then the comment is really confusing.